### PR TITLE
Add CentOS 7.2 for OpenStack

### PR DESCRIPTION
**Pull request by zmc**

### CODE REVIEW

**Comment by zmc** [comment]
The user-data file is identical to 7.1's

File: teuthology/openstack/__init__.py [hunk 1]
```diff
@@ -181,8 +181,9 @@ class OpenStack(object):
     # wget -O fedora-20.qcow2 http://fedora.mirrors.ovh.net/linux/releases/20/Images/x86_64/Fedora-x86_64-20-20131211.1-sda.qcow2
     image2url = {
         'centos-6.5': 'http://cloud.centos.org/centos/6/images/CentOS-6-x86_64-GenericCloud-1508.qcow2',
-        'centos-7.0': 'http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1508.qcow2',
-        'centos-7.1': 'http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1508.qcow2',
+        'centos-7.0': 'http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1503.qcow2',
```
Comment (andrewschoen) [inline teuthology/openstack/__init__.py]: just double checking, are both 7.0 and 7.1 the same image?

Comment (zmc) [inline teuthology/openstack/__init__.py]: Yeah, because 1503 / 7.1 is actually the oldest they provide :-/

Comment (andrewschoen) [inline teuthology/openstack/__init__.py]: Ah, ok. Just wanted to make sure it wasn't a typo or something.

**Pull request merged**